app/products/playbooks/client/rest.test.ts

```diff
@@ -437,3 +515,490 @@ describe('finishRun', () => {
+        const name = 'Test Run';
+        const description = 'Test Description';
+        const channel_id = 'channel123';
+        const expectedUrl = '/plugins/playbooks/api/v0/runs';
+        const expectedOptions = {
+            method: 'post',
+            body: {
+                owner_user_id,
+                team_id,
+                name,
+                description,
+                playbook_id,
+                channel_id,
+                create_public_run: undefined,
+            },
+        };
+        const mockResponse = {
+            id: 'run123',
+            name,
+            playbook_id,
+        };
+
+        jest.mocked(client.doFetch).mockResolvedValue(mockResponse);
+
+        const result = await client.createPlaybookRun(
+            playbook_id,
+            owner_user_id,
+            team_id,
+            name,
+            description,
+            channel_id,
+        );
+
+        expect(client.doFetch).toHaveBeenCalledWith(expectedUrl, expectedOptions);
+        expect(result).toEqual(mockResponse);
+    });
+
+    test('should create playbook run with create_public_run only', async () => {
+        const playbook_id = 'playbook123';
+        const owner_user_id = 'user123';
+        const team_id = 'team123';
+        const name = 'Test Run';
+        const description = 'Test Description';
+        const create_public_run = false;
+        const expectedUrl = '/plugins/playbooks/api/v0/runs';
+        const expectedOptions = {
+            method: 'post',
+            body: {
+                owner_user_id,
+                team_id,
+                name,
+                description,
+                playbook_id,
+                channel_id: undefined,
+                create_public_run,
+            },
+        };
+        const mockResponse = {
+            id: 'run123',
+            name,
+            playbook_id,
+        };
+
+        jest.mocked(client.doFetch).mockResolvedValue(mockResponse);
+
+        const result = await client.createPlaybookRun(
+            playbook_id,
+            owner_user_id,
+            team_id,
+            name,
+            description,
+            undefined,
+            create_public_run,
+        );
+
+        expect(client.doFetch).toHaveBeenCalledWith(expectedUrl, expectedOptions);
+        expect(result).toEqual(mockResponse);
+    });
+
+    test('should handle error when creating playbook run', async () => {
+        const playbook_id = 'playbook123';
+        const owner_user_id = 'user123';
+        const team_id = 'team123';
+        const name = 'Test Run';
+        const description = 'Test Description';
+
+        jest.mocked(client.doFetch).mockRejectedValue(new Error('Network error'));
+
+        await expect(
+            client.createPlaybookRun(playbook_id, owner_user_id, team_id, name, description),
+        ).rejects.toThrow('Network error');
+    });
+
+    test('should create playbook run with empty strings', async () => {
```

Comment:
is this one right? 
some id's I can see them being blank, but not sure about the `team` one or `name`. While it might be technically possible, is there a reason for accepting it?

Reply:
This is one of those tests that AI likes to do, and I forgot to remove.

The client function is not responsible of this right now. 0/5 whether it should be responsible.

Reply:
feel free to remove it